app-5-rdf-serializer/src/main/java/org/geovistory/toolbox/streams/rdf/processors/project/ProjectCustomRdfsLabels.java

```diff
@@ -0,0 +1,112 @@
+package org.geovistory.toolbox.streams.rdf.processors.project;
+
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.kstream.KStream;
+import org.apache.kafka.streams.kstream.Produced;
+import org.geovistory.toolbox.streams.avro.*;
+import org.geovistory.toolbox.streams.rdf.AvroSerdes;
+import org.geovistory.toolbox.streams.rdf.OutputTopicNames;
+import org.geovistory.toolbox.streams.rdf.RegisterInputTopic;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.geovistory.toolbox.streams.lib.UrlPrefixes.*;
+
+
+@ApplicationScoped
+public class ProjectCustomRdfsLabels {
+
+
+    @Inject
+    AvroSerdes avroSerdes;
+
+    @Inject
+    RegisterInputTopic registerInputTopic;
+
+
+    @Inject
+    OutputTopicNames outputTopicNames;
+
+    public ProjectCustomRdfsLabels(AvroSerdes avroSerdes, RegisterInputTopic registerInputTopic, OutputTopicNames outputTopicNames) {
+        this.avroSerdes = avroSerdes;
+        this.registerInputTopic = registerInputTopic;
+        this.outputTopicNames = outputTopicNames;
+    }
+
+    public void addProcessorsStandalone() {
+         addProcessors(
+                registerInputTopic.projectStream()
+        );
+    }
+
+    public ProjectRdfReturnValue addProcessors(
+            KStream<ProjectRdfKey, ProjectRdfValue> projectStream
```

Comment:
The input stream is KStream<dev.projects.project.Key, dev.projects.project.Value>

Reply:
Done